corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
+    template_name = 'data_interfaces/interfaces/import_forms.html'
+    urlname = 'archive_forms'
+    page_title = ugettext_noop("Archive Forms")
+
+    @property
+    def page_url(self):
+        return reverse(self.urlname, args=[self.domain])
+
+    @property
+    def page_context(self):
+        context = {}
+        context.update({
+            'bulk_upload': {
+                "download_url": static(
+                    'data_interfaces/files/forms_bulk_example.xlsx'),
+                "adjective": _("example"),
+                "verb": _("archive"),
+                "plural_noun": _("forms"),
+            },
+        })
+        context.update({
+            'bulk_upload_form': get_bulk_upload_form(context),
+        })
+        return context
+
+    @property
+    @memoized
+    def uploaded_file(self):
+        try:
+            bulk_file = self.request.FILES['bulk_upload_file']
+        except KeyError:
+            raise BulkUploadCasesException(_("No files uploaded"))
+        try:
+            return WorkbookJSONReader(bulk_file)
+        except InvalidFileException:
+            try:
+                csv.DictReader(io.StringIO(bulk_file.read().decode('ascii'),
```

Comment:
This was just precedent. I looked at other bulk uploaders in the code base and saw the use of ascii. I agree UTF8 is better and will change